src/assertions/there.js

```diff
@@ -0,0 +1,21 @@
+import elementExists from '../util/element-exists';
+
+export default function there(client, chai, utils) {
```

Comment:
I wanted to call this `exist`, but ... I think that was conflicting with http://chaijs.com/api/bdd/#method_exist so I had to call it `there`

I suppose maybe that's why chai-webdriver used the `dom` flag?  (To be able to provide alternate behavior for some of the predefined assertions?)  Anyway, if you have a better idea for a name, or for how to work with this, let me know!